RockMigrations.module.php

```diff
@@ -1643,18 +1643,18 @@ public function getModuleConfig($module) {
      *
      * @param string $name
      * @param string $url
-     * @return void
+     * @return Module
      */
     public function installModule($name, $url = null) {
       // if the module is already installed we return it
-      $module = $this->modules->get((string)$name);
+      $module = $this->modules->get((string)$name); // this is not good because getting a module will also install it, so if the files are on disk they are installed
```

Comment:
Thx for the hint, but we are in the installModule() method, so we actually want to install the module anyhow?!

Reply:
Yes, if you also provide a URL, and a module with the name is already present on disk, the download would never happen. Imagine, you would like to provide a different URL with another version/branch of the module, but the same module name.